hilti/include/base/util.h

```diff
@@ -408,6 +408,15 @@ using hilti::rt::escapeBytes;
 using hilti::rt::escapeUTF8;
 using hilti::rt::expandEscapes;
 
+/**
+ * Wrapper for `escapeBytes` that produces a valid C++ string literal.
+ *
+ * @param str string to escape
+ * @return escaped std::string (without the quotes)
```

Comment:
The `without the quotes` part made me pause. Are you talking about how literals are formatted in (input) C++ source code? -- in that case it should be clear that these quotes are not part of the literal itself, and I wouldn't expect them to be preserved when calling anything but possibly very arcane macros.
```suggestion
 * @return escaped std::string
```